src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -652,66 +652,6 @@ function copyOptions() {
     feedbackRecipientUpdateVisibilityOptions($currRecipient);
 }
 
-function enableRow(elem, row) {
-    var $visibilityOptions = $(elem).closest('form').find('.visibilityOptions');
-    var $table = $visibilityOptions.find('table');
-    var $tdElements = $($table.children().children()[row]).children();
-    
-    if ($tdElements.parent().prop('tagName') === 'tr') {
-        return;
-    }
-    $tdElements.unwrap().wrapAll('<tr>');
-}
-
-function disableRow(elem, row) {
-    var $visibilityOptions = $(elem).closest('form').find('.visibilityOptions');
-    var $table = $visibilityOptions.find('table');
-    var $tdElements = $($table.children().children()[row]).children();
-    
-    if ($tdElements.parent().prop('tagName') === 'hide') {
-        return;
-    }
-    $tdElements.unwrap().wrapAll('<hide>');
-    $tdElements.parent().hide();
-}
-
-function feedbackRecipientUpdateVisibilityOptions(elem) {
```

Comment:
I left these in `instructorFeedbackEdit.js` because I was actually intending to rationalize these and move some of them to `feedbackPath.js` (as in #5604) because the event is triggered on the Giver/Recipient input. (I did moved `isRecipientsTeamMembersVisibilityOptionInvalidForRecipientType` over to `visibilityOptions.js` because it clearly belongs there)